runtime/android/core_internal/src/org/xwalk/core/internal/XWalkViewInternal.java

```diff
@@ -941,6 +941,18 @@ void setOverlayVideoMode(boolean enabled) {
         mContent.setOverlayVideoMode(enabled);
     }
 
+    /**
+    * Control whether the XWalkView's surface is placed on top of its window.
+    * Note this only works when XWalkPreferences.ANIMATABLE_XWALK_VIEW is false.
+    * @param onTop true for on top.
+    * @since 5.0
+    */
+    @XWalkAPI
+    public void setZOrderOnTop(boolean onTop) {
+        if (mContent == null) return;
+        mContent.setZOrderOnTop(onTop);
```

Comment:
Yes, I suggest other places change as this style. XWalkContent.java and chromium part as well.